lib/cmd/common/version.js

```diff
@@ -7,7 +7,7 @@ const PATTERN = /\+BUILD-NUMBER/;
 var fhc = require('../../fhc'),
   ini = require('../../utils/ini'),
   _ = require('underscore'),
-  request = require('request').defaults({'proxy': fhc.config.get("proxy"), timeout: timeout}),
+  request = require('request'),   // don't set proxy or other configured defaults here, since the config may not be loaded when this module is
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingComment)